subgraph-js/src/index.ts

```diff
@@ -1,2 +1 @@
 export { buildSubgraphSchema } from './buildSubgraphSchema';
-export { printSubgraphSchema } from './printSubgraphSchema';
```

Comment:
**question** Was this only used for testing? Or are we suggesting using the `graphql-js` version?

Reply:
I wasn't sure if this was an API we thought users would use, no need to re-add it if we don't think it was used outside of `subgraph-js`.

Reply:
you know, i literally just spoke to a user who's using it. granted, they were using it to fix a bug that this pr fixes. but maybe let's keep it around just in case